src/Mailgun/Exception/HttpClientException.php

```diff
@@ -70,7 +70,7 @@ public static function requestFailed(ResponseInterface $response = null)
 
     public static function notFound(ResponseInterface $response = null)
     {
-        return new self('The endpoint you tried to access does not exist. Check your URL.', 404, $response);
+        return new self('The endpoint you have tried to access does not exist. Check if the domain matches the domain you have configure on Mailgun.', 404, $response);
```

Comment:
@Nyholm do you really think is better to have the same copy twice, that means increased maintenance cost due we need to change the same copy twice?